integration/load.php

```diff
@@ -162,3 +162,15 @@ function ( $post_types ) {
  * @see https://buddypress.org/
  */
 add_action( 'bp_include', array( __NAMESPACE__ . '\Buddypress', 'init' ), 0 );
+
+/**
+ * Load the Surge integration.
+ *
+ * Only load code that needs Surge to run once Surge is loaded and initialized.
+ *
+ * @see https://wordpress.org/plugins/surge/
+ */
+Surge::init();
```

Comment:
Sorry, I meant this could be called with the other integrations in `plugin_init()`.